### PR TITLE
Lmb 686/banner for environment

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

We want to show a banner when we are in a different environment.

## How to test

Forcefully change the enviroment name to DEV, QA, PROD, LOCAL

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/215

## Attachments

image

### CODE REVIEW

File: app/controllers/application.js [hunk 1]
```diff
@@ -35,4 +36,73 @@ export default class ApplicationController extends Controller {
       this.setNotificationCount.perform();
     }
   });
+
+  get environmentMessage() {
+    const productieUrl = 'https://mandatenbeheer.lokaalbestuur.vlaanderen.be/';
+    const applicationName = 'Lokaal Mandatenbeheer';
```
Comment (JonasVanHoof) [inline app/controllers/application.js:42]: This can come out of the environments if we want

File: config/environment.js [hunk 2]
```diff
@@ -6,6 +6,22 @@ module.exports = function (environment) {
...
+        enabled: '{{ANNOUNCE_NEW_DEPLOYMENT_ENABLED}}',
+        message: '{{ANNOUNCE_NEW_DEPLOYMENT_MESSAGE}}',
+      },
+      testing: {
+        enabled: '{{ANNOUNCE_TESTING_ENABLED}}',
+        message: '{{ANNOUNCE_TESTING_MESSAGE}}',
+      },
+    },
+    appName: 'Lokaal Mandatenbeheer',
+    environmentName: '{{ENVIRONMENT_NAME}}',
```
Comment (Rahien) [inline config/environment.js:24]: could you create a pr that sets this in the app?

Comment (JonasVanHoof) [reply config/environment.js:9]: Pr for setting the environment name in the LMB app https://github.com/lblod/app-lokaal-mandatenbeheer/pull/215

**Review by bfidlers — APPROVED** [review]
Works as expected!

**Comment by bfidlers** [comment]
You'll have to fix the pipeline, because it can't do the `npm install --legacy-peer-deps`

**Pull request merged**